commit e33d5e4cfb025f19628cb1e57e59b115af58da5a
Author: Zach Kuznia <zork@chromium.org>
Date:   Wed Apr 28 17:19:59 2010 -0700

    Patch in fix to chewing_zuin_String.
    Add README.chromium
    
    BUG=none
    TEST=none
    
    Review URL: http://codereview.chromium.org/1701019

diff --git a/src/mod_aux.c b/src/mod_aux.c
index 12936a1..d6d8914 100644
--- a/src/mod_aux.c
+++ b/src/mod_aux.c
@@ -90,7 +90,7 @@ CHEWING_API char *chewing_zuin_String( ChewingContext *ctx, int *zuin_count )
 		if ( ctx->output->zuinBuf[ i ].s[ 0 ] != '\0' ) {
 			strcat( s, (char *) (ctx->output->zuinBuf[ i ].s) );
 			if ( zuin_count )
-				*zuin_count++;
+				(*zuin_count)++;
 		}
 	}
 	return s;
